### ISSUE TITLE
UnauthorizedAccessException on every attempted run

### ISSUE THREAD

**Issue by CryptArchy** [OpenCover/opencover#426]
I pulled OpenCover down through Nuget and attempted to run it against our application, but no matter what options I input, I get the same UnauthorizedAccessException.  I've tried running in an admin console, running the console program itself as admin, putting the commands in a .bat file and running that as admin.  Nothing. 

I'm on Windows 8.1, using VS2015 and the latest version of xUnit.  Here is the output error:

```
C:\Users\<me>\Code\...\packages\OpenCover.4.6.166\tools>  "C:\Users\<me>\Code\...\packages\OpenCover.4.6.166\tools\OpenCover.Console.exe" -target:"C:\Users\<me>\Code\...\packages\xunit.runner.console.2.1.0\tools\xunit.console.exe" -output :"C:\Users\<me>\Code\...\Coverage.xml" -targetargs:"C:\Users\<me>\Code\...\Tests\bin\Debug\Tests.dll -NUnit C:\Users\<me>\Code\...\TestResults-Nunit.xml -XML C:\Users\<me>\Code\...\TestResults.xml" -register:user

...
=== TEST EXECUTION SUMMARY ===
Tests  Total: 12, Errors: 0, Failed: 0, Skipped: 0, Time: 3.325s
Committing...

Unhandled Exception: System.UnauthorizedAccessException: Access to the path 'C:\Users\<me>\Code\...\packages\OpenCover.4.6.166\tools' is denied.
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, String msgPath, Boolean bFromProxy)
   at System.IO.FileStream..ctor(String path, FileMode mode)
   at OpenCover.Framework.Persistance.FilePersistance.SaveCoverageFile() in c:\projects\opencover\main\OpenCover.Framework\Persistance\FilePersistance.cs:line 83
   at OpenCover.Framework.Manager.ProfilerManager.<>c__DisplayClass7.<SaveVisitData>b__6(Object state) in c:\projects\opencover\main\OpenCover.Framework\Manager\ProfilerManager.cs:line 154
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   at System.Threading.QueueUserWorkItemCallback.System.Threading.IThreadPoolWorkItem.ExecuteWorkItem()
   at System.Threading.ThreadPoolWorkQueue.Dispatch()
```

**Comment by molnargab** [comment]
There's a space before the colon in the command you posted, at `-output :"C:\Users\<me>\Code\...\Coverage.xml"`. This may be causing this issue.

**Comment by CryptArchy** [comment]
Oh my goodness, yes, that was the issue!  I'm sorry for the false report.  Would be nice if this mistake on my part was more obvious from the error message, though.

**Comment by molnargab** [comment]
The command-line parser has already been patched (as part of #335) to fail on unknown arguments (such as `:"C:\Users\<me>\Code\...\Coverage.xml"`). The pre-release builds should already have it.

**Comment by sawilde** [comment]
I think I am okay to close this issue.
